17-lifecycle-lab/src/02-cps-with-indicator.jsx

```diff
@@ -1,26 +1,60 @@
-/**
- * Lab 02: CPS monitor with indicator
- *
- * Add an indicator panel to the CPS monitor
- * that shows a green rectangle if user clicks fast enough
- * or a red rectangle if user clicks too slow
- *
- * Decide how to share information between the components
- * and what values to pass when creating the Indicator
- */
-
-import React from 'react';
-
-const Indicator = React.createClass({
-});
-
-export default createClass({
-  render() {
-    return (<div>
-      <button>Click Fast</button>
-      <p>CPS rate: {0}</p>
-      <Indicator />
-    </div>);
-  }
-});
-
+import React from 'react';
+
+export default class ClicksCounter extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            counter: 0,
+            fastPlayer: false,
+        };
+
+        this.incClickCounter = this.incClickCounter.bind(this);
+    }
+
+    componentWillMount() {
+        this.timer = setInterval(() => {
+            this.setState({counter: 0});

```

Comment:
Look at the comment of exercise 1 - this is not the real cps. Please modify first two exercises accordingly